docs/section19/19-19.md

```diff
@@ -0,0 +1,94 @@
+            - Reset total exhaust cfm: `total_exhaust_cfm = 0`       
+            - Reset total relief cfm: `total_relief_cfm = 0`       
+            - Get the fan system object:  `fan_sys_b = hvac.fan_system`  
+            - Get list of supply fan fan objects: `fan_sys_supply_list = list(fan_sys_b.supply_fans)`  
+            - Get list of return fan fan objects: `fan_sys_return_list = list(fan_sys_b.return_fans)`
+            - Get list of relief fan fan objects: `fan_sys_relief_list = list(fan_sys_b.relief_fans)`
+            - Get list of exhaust fan fan objects: `fan_sys_exhaust_list = list(fan_sys_b.exhaust_fans)`
+                    
+            Below cycles through all fans associated with the HVAC system and sums the total fan kW.
+            - Cycle through the list of supply fans: `for fan_b in fan_sys_supply_list:`  
+                - Get the supply cfm and add to cumulative supply cfm as the logic cycles through the supply fans: `supply_cfm_b = supply_cfm_b + fan_b.design_airflow`  
+                - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,fan_b)`    
+                - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`
+            - Cycle through the list of return fans: `for fan_b in fan_sys_return_list:`  
+                - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,fan_b)`    
+                - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`
+            - Cycle through the list of relief fans: `for fan_b in fan_sys_relief_list:`  
+                - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,fan_b)`    
+                - Get the cfm: `total_relief_cfm =  total_relief_cfm + fan_b.design_airflow` 
+                - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`       
+            - Cycle through the list of exhaust fans: `for fan_b in fan_sys_exhaust_list:`  
+                - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,fan_b)`   
+                - Get the cfm: `total_exhaust_cfm =  total_exhaust_cfm + fan_b.design_airflow`  
+                - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`     
+            - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                    - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                    - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`                    
```

Comment:
You ahve this routine in several RDSs... perhaps it should be a function? No strong feelings since you already implemented it.

Reply:
Done